packages/firebase_auth/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -609,18 +609,31 @@ class FirebaseAuth extends FirebasePluginPlatform {
     }
   }
 
-  /// Signs in with an AuthProvider using native authentication flow.
+  /// Signs in with an AuthProvider using native authentication flow. This is
+  /// deprecated in favour of calling directly `signInWithProvider()`.
```

Comment:
```suggestion
  /// deprecated in favor of `signInWithProvider()`.
```

American english for consistency 😅

Reply:
I copy and pasted from somewhere else in the code, I'll change it here too 👍